src/main/java/org/sagebionetworks/web/client/widget/doi/CreateOrUpdateDoiModalView.java

```diff
@@ -32,7 +32,7 @@
 
 	void setPublicationYear(Long publicationYear);
 
-	void showOverwriteWarning(boolean showWarning);
+	void setOverwriteWarningVisible(boolean showWarning);
```

Comment:
No change, just clarity/consistency with other view visibility methods

Reply:
(could rename showWarning to visible)